app/templates/home.html

```diff
@@ -4,8 +4,10 @@
 
 {% block main_column %}
 
-  {% if error %}
-    <p>{{ error }}</p>
+  {% if not current_user.is_superuser %}
+  <h2 class="heading-medium">No access yet</h2>
```

Comment:
As discussed in another PR, I always prefer to indent in the templates to make them more readable and avoid confusion/bugs.

Reply:
Yep, I knew you would say that - I will change